crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -263,14 +267,22 @@ impl FromWorld for Mesh2dPipeline {
         )> = SystemState::new(world);
         let (render_device, render_queue, default_sampler) = system_state.get_mut(world);
         let render_device = render_device.into_inner();
+        let tonemapping_lut_entries = get_lut_bind_group_layout_entries();
         let view_layout = render_device.create_bind_group_layout(
             "mesh2d_view_layout",
-            &BindGroupLayoutEntries::sequential(
+            &BindGroupLayoutEntries::with_indices(
```

Comment:
Why not keep this sequential?

Reply:
From what I read in the Bevy Discord, the common consensus is that it's better to use explicit indices instead of sequential